android/src/main/java/com/mattermost/networkclient/NetworkClient.kt

```diff
@@ -61,6 +76,18 @@ class NetworkClient(private val baseUrl: HttpUrl? = null, private val options: R
         }
     }
 
+    fun importClientP12(p12FilePath: String, password: String) {
+        val handshakeCertificates = getHandshakeCertificates(p12FilePath, password)
+        if (handshakeCertificates != null) {
+            okHttpClient = okHttpClient.newBuilder()
```

Comment:
Would doing it this way still work for the swizzling? Just wondering if this client will be available via the RCT interceptor.